src/__tests__/shared/components/input/DateTimePickerWithLabelFormGroup.test.tsx

```diff
@@ -51,23 +51,23 @@ describe('date picker with label form group', () => {
       expect(datepickerInput).toHaveDisplayValue(/[12/25/2020 2:56 PM]/i)
     })
   })
+})
 
-  describe('change handler', () => {
-    it('should call the change handler on change', () => {
-      render(
-        <DateTimePickerWithLabelFormGroup
-          name="stardate55555555"
-          label="stardate55555555"
-          value={new Date('1/1/2021 2:56 PM')}
-          isEditable
-          onChange={jest.fn()}
-        />,
-      )
-      const datepickerInput = screen.getByRole('textbox')
+describe('change handler', () => {
+  it('should call the change handler on change', () => {
+    render(
+      <DateTimePickerWithLabelFormGroup
+        name="stardate55555555"
+        label="stardate55555555"
+        value={new Date('1/1/2021 2:56 PM')}
+        isEditable
+        onChange={jest.fn()}
```

Comment:
I liked what you did in the other PR I just merged with a `TestComponent` and a `useState` to test the change handler is being called.

Reply:
yeah, this one somehow passes even without that 🤷

Reply:
hmm, but the warning is still present when I re-run the test again...

I have to investigate this test more, closing PR for now :(